src/components/templates/PartyMap/components/RoomAttendance.tsx

```diff
@@ -15,20 +15,13 @@ interface PropsType {
 const MAX_AVATARS_VISIBLE = 2;
 
 export const RoomAttendance: FC<PropsType> = ({ venue, room }) => {
-  const partygoers = usePartygoers();
-  const usersInRoom = useMemo(
-    () =>
-      partygoers?.filter(
-        (partygoer) => partygoer.lastSeenIn?.[`${venue.name}/${room.title}`]
-      ),
-    [partygoers, venue.name, room.title]
-  );
+  const { recentRoomUsers } = useRecentRoomUsers(room.title);
 
-  const numberOfUsersInRoom = usersInRoom?.length;
+  const numberOfUsersInRoom = recentRoomUsers.length;
   if (numberOfUsersInRoom < 1) return <></>;
```

Comment:
```suggestion

  if (numberOfUsersInRoom < 1) return null;

```

Reply:
Fixed this, but please let's stay in the scope of this PR

Reply:
I will always call out issues I see in the code, regardless of whether they were introduced in the PR, or you just changed nearby code.

The effort of back logging this + fixing it in a separate PR far outweighs the cost of fixing it here.

if it was something more substantial, then it would make sense to backlog it. But I would still call it out when I saw it in my review.

Thanks for fixing it.